addon/ng2/blueprints/ng2/files/__path__/test.ts

```diff
@@ -0,0 +1,41 @@
+
+/*global jasmine, __karma__, window*/
+require('core-js/es6');
+require('core-js/es7/reflect');
+
+// Typescript emit helpers polyfill
+require('ts-helpers');
```

Comment:
this is coming from TS 2.0 anyway, and its one more random dependency to worry about. see https://github.com/Microsoft/TypeScript/pull/9097 for the correct version of how this should be done, for the moment we should use the built-in behavior from TS.